### PR TITLE
Add new SWA option ADMIN_SETS_CREDENTIALS to Authentication Schemes in Apps

### PR DESCRIPTION
**Pull request by sahilshah-okta**
## Changes
- Adds new SWA scheme ADMIN_SETS_CREDENTIALS to the list of existing Authentication schemes.
- Adds new Credential Operation "Set Plugin SWA to Administrator Sets Username and Password"
to Apps(Okta API) collection for Postman

## Snapshots

screen shot 2015-11-24 at 2 45 41 pm

screen shot 2015-11-24 at 3 22 34 pm

screen shot 2015-11-24 at 2 29 43 pm

[OKTA-70799](https://oktainc.atlassian.net/browse/OKTA-70799)

### CODE REVIEW

**Comment by sahilshah-okta** [comment]
@intinfra-okta @joelfranusic-okta @karlmcguinness-okta Could you please review? [OKTA-70799](https://oktainc.atlassian.net/browse/OKTA-70799)

File: _source/docs/api/resources/apps.md [hunk 1]
```diff
@@ -2000,6 +2001,123 @@ curl -v -X PUT \
...
+    "userNameTemplate": {
+      "template": "${source.login}",
+      "type": "BUILT_IN"
+    }
+  },
+  "settings": {
+    "app": {
+      "buttonField": "btn-login",
+      "passwordField": "txtbox-password",
+      "usernameField": "txtbox-username",
```
Comment (joelfranusic-okta) [inline _source/docs/api/resources/apps.md]: I'm guessing that the `buttonField`, `passwordField`, and `usernameField` are the HTML "id" field names to be used by the SWA plugin to enter credentials? Is that correct? If so, could we update the example strings above to something like `id-attribute-name-for-submit-button`, `id-attribute-name-for-username-field`, and `id-attribute-name-for-password-field`?

File: _source/docs/api/resources/apps.md [hunk 2]
```diff
@@ -2000,6 +2001,123 @@ curl -v -X PUT \
...
+      "url": "https://example.com/login.html"
```
Comment (joelfranusic-okta) [inline _source/docs/api/resources/apps.md]: Nice use of example.com here. Thanks!

File: _source/docs/api/resources/apps.md [hunk 3]
```diff
@@ -2000,6 +2001,123 @@ curl -v -X PUT \
...
+    "userNameTemplate": {
+      "template": "${source.login}",
+      "type": "BUILT_IN"
+    }
+  },
+  "settings": {
+    "app": {
+      "buttonField": "btn-login",
+      "passwordField": "txtbox-password",
+      "usernameField": "txtbox-username",
```
Comment (joelfranusic-okta) [inline _source/docs/api/resources/apps.md]: Same comments as above.

**Comment by joelfranusic-okta** [comment]
Generally, this looks good to me. I have a stylistic suggestion on the naming of the example text, but I'd be okay with shipping this update as is.

File: _source/docs/api/resources/apps.md (hunk 1 above)
Comment (sahilshah-okta) [inline _source/docs/api/resources/apps.md]: All existing SWA authentication schemes had these values for the above fields, hence I chose to be consistent with them.

**Pull request merged**